### PR TITLE
Fix build on master 🔧💚

### LINKED ISSUE #156: Master fails to build
After merging in https://github.com/interchainio/tendermint-rs/pull/146 master fails to build. Although github didn't show any merge conflicts this due to the change in error handling. I'll fix in a bit!

This is sth we should keep in mind with other PRs related to the light client.

### PR DESCRIPTION
**Pull request by Liamsi**
closes #156

* [ ] Referenced an issue explaining the need for the change
* [ ] Updated all relevant documentation in docs
* [ ] Updated all code comments where relevant
* [ ] Wrote tests
* [ ] Updated CHANGES.md

### CODE REVIEW

**Comment by Shivani912** [comment]
👍

File: tendermint/tests/lite.rs [hunk 1]
```diff
@@ -82,7 +84,7 @@ impl Requester<SignedHeader, Header> for MockRequester {
             return Ok(sh.into());
         }
         println!("couldn't get sh for: {}", &h);
-        Err(Error::RequestFailed)
+        fail!(Kind::RequestFailed, "couldn't get sh for: {}", &h);
```
Comment (Liamsi) [inline tendermint/tests/lite.rs]: This is just a shortcut for 
```rust
return Kind::Kind::RequestFailed.context(format!("couldn't get sh for: {}", &h)).into();
```

File: tendermint/tests/lite.rs [hunk 2]
```diff
@@ -1,10 +1,12 @@
+use anomaly::fail;
 use serde::{de::Error as _, Deserialize, Deserializer, Serialize};
 use serde_json;
 use std::collections::HashMap;
 use std::convert::TryInto;
 use std::{fs, path::PathBuf};
 use tendermint::block::{Header, Height};
-use tendermint::lite::{Error, Requester, TrustThresholdFraction, TrustedState};
```
Comment (Liamsi) [inline tendermint/tests/lite.rs]: Errors moved into their own module as per #149

File: tendermint/tests/lite.rs (hunk 1 above)
Comment (Shivani912) [reply tendermint/tests/lite.rs]: Okay, I was close but didn't know about this shortcut. Thanks!

**Pull request merged**